### PR TITLE
Update to Outdated files in dev-1.20-ko.5 branch (3)

### PR DESCRIPTION
**Pull request by ysyukr**
Fixes the part of #26620

This PR contains updates for the following files:
- [x] M51. content/en/docs/tasks/administer-cluster/change-default-storage-class.md
- [ ] M52. content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md | applied to #26262
- [x] M53. content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md
- [x] M54. content/en/docs/tasks/configure-pod-container/static-pod.md
- [x] M55. content/en/docs/tasks/extend-kubectl/kubectl-plugins.md
- [x] M56. content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md
- [x] M57. content/en/docs/tasks/job/parallel-processing-expansion.md
- [x] M58. content/en/docs/tasks/run-application/delete-stateful-set.md
- [x] M59. content/en/docs/tasks/run-application/horizontal-pod-autoscale.md
- [x] M60. content/en/docs/tasks/run-application/run-single-instance-stateful-application.md
- [x] M61. content/en/docs/tutorials/_index.md
- [x] M62. content/en/docs/tutorials/clusters/apparmor.md
- [x] M63. content/en/docs/tutorials/kubernetes-basics/_index.html
- [x] M64. content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html
- [x] M65. content/en/docs/tutorials/stateful-application/basic-stateful-set.md
- [x] M66. content/en/docs/tutorials/stateful-application/zookeeper.md
- [x] M67. content/en/docs/tutorials/stateless-application/guestbook.md
- [x] M68. content/en/examples/application/guestbook/frontend-deployment.yaml
- [x] M69. content/en/examples/application/guestbook/frontend-service.yaml
- [x] D1. content/en/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana.md
- [x] D2. content/en/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md
- [x] D3. content/en/examples/application/guestbook/redis-master-deployment.yaml
- [x] D4. content/en/examples/application/guestbook/redis-master-service.yaml
- [x] D5. content/en/examples/application/guestbook/redis-slave-deployment.yaml
- [x] D6. content/en/examples/application/guestbook/redis-slave-service.yaml

This list is a file added by calling from a specific document.
- [x] content/ko/examples/application/guestbook/mongo-deployment.yaml
- [x] content/ko/examples/application/guestbook/mongo-service.yaml

### CODE REVIEW

File: content/ko/docs/tasks/administer-cluster/change-default-storage-class.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ content_type: task
 수도 있다. 이런 경우에, 기본 스토리지 클래스를 변경하거나 완전히 비활성화
 하여 스토리지의 동적 프로비저닝을 방지할 수 있다.
 
-단순하게 기본 스토리지클래스를 삭제하는 경우, 사용자의 클러스터에서 구동중인
+기본 스토리지클래스를 삭제하는 경우, 사용자의 클러스터에서 구동중인
```
Comment (pjhwa) [inline content/ko/docs/tasks/administer-cluster/change-default-storage-class.md:35]:
```suggestion
기본 스토리지클래스를 삭제하는 경우, 사용자의 클러스터에서 구동 중인
```

- 띄어쓰기 수정합니다.
* 참고로, 원문의 업데이트를 반영할 때 이런 류(띄어쓰기, 오타, 오역 등)가 눈에 띄면 바로 수정할 지, 다른 PR로 수정할 지 고민이 되더라고요.

File: content/ko/docs/tasks/configure-pod-container/static-pod.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ Kubelet 은 각각의 스태틱 파드에 대하여 쿠버네티스 API 서버
 생성하려고 자동으로 시도한다.
 즉, 노드에서 구동되는 파드는 API 서버에 의해서 볼 수 있지만,
 API 서버에서 제어될 수는 없다.
+파드 이름에는 선두의 하이픈과 함께 노드 호스트 이름이 접미사로 추가 된다.
```
Comment (pjhwa) [inline content/ko/docs/tasks/configure-pod-container/static-pod.md:25]:
```suggestion
파드 이름에는 노드 호스트 이름 앞에 하이픈을 붙여 접미사로 추가된다.
```

- 좀더 이해하기 쉽게 수정했습니다.

File: content/ko/docs/tasks/extend-kubectl/kubectl-plugins.md [hunk 3]
```diff
@@ -57,9 +57,9 @@ Krew [플러그인 인덱스](https://krew.sigs.k8s.io/plugins/)를 통해 사
 
 플러그인 설치 또는 사전 로딩이 필요하지 않다. 플러그인 실행 파일은
 `kubectl` 바이너리에서 상속된 환경을 받는다.
-플러그인은 이름을 기반으로 구현할 명령 경로를 결정한다. 예를
-들어, 새로운 명령인 `kubectl foo` 를 제공하려는 플러그인은 단순히 이름이
-`kubectl-foo` 이고, `PATH` 의 어딘가에 있다.
+플러그인은 이름을 기반으로 구현할 명령 경로를 결정한다.
+예를 들어, `kubectl-foo` 라는 플러그인은 `kubectl foo` 명령을 제공한다. 이를
+`PATH` 어딘가에 플러그인 실행 파일을 설치해야 한다.
```
Comment (pjhwa) [inline content/ko/docs/tasks/extend-kubectl/kubectl-plugins.md:62]:
```suggestion
예를 들어, `kubectl-foo` 라는 플러그인은 `kubectl foo` 명령을 제공한다.
`PATH` 어딘가에 플러그인 실행 파일을 설치해야 한다.
```

- '이를'이 앞에 붙어 어색한 것 같아 삭제했습니다.

File: content/ko/docs/tasks/run-application/delete-stateful-set.md [hunk 4]
```diff
@@ -60,7 +60,7 @@ PVC를 삭제할 때 데이터 손실될 수 있음에 주의하자.
 
 ### 스테이트풀셋의 완벽한 삭제
 
-연결된 파드를 포함해서 스테이트풀셋의 모든 것을 간단히 삭제하기 위해 다음과 같이 일련의 명령을 실행 한다.
+연결된 파드를 포함해서 스테이트풀셋의 모든 것을 삭제하기 위해 다음과 같이 일련의 명령을 실행 한다.
```
Comment (pjhwa) [inline content/ko/docs/tasks/run-application/delete-stateful-set.md:63]:
```suggestion
연결된 파드를 포함해서 스테이트풀셋의 모든 것을 삭제하기 위해 다음과 같이 일련의 명령을 실행한다.
```

File: content/ko/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 5]
```diff
@@ -190,7 +190,7 @@ Horizontal Pod Autoscaler는 모든 API 리소스와 마찬가지로 `kubectl`
 `kubectl get hpa`로 오토스케일러 목록을 조회할 수 있고, `kubectl describe hpa`로 세부 사항을 확인할 수 있다.
 마지막으로 `kubectl delete hpa`를 사용하여 오토스케일러를 삭제할 수 있다.
 
-또한 Horizontal Pod Autoscaler를 쉽게 생성 할 수 있는 `kubectl autoscale`이라는 특별한 명령이 있다.
+또한 Horizontal Pod Autoscaler를 생성 할 수 있는 `kubectl autoscale`이라는 특별한 명령이 있다.
```
Comment (pjhwa) [inline content/ko/docs/tasks/run-application/horizontal-pod-autoscale.md:193]:
```suggestion
또한 Horizontal Pod Autoscaler를 생성할 수 있는 `kubectl autoscale`이라는 특별한 명령이 있다.
```

File: content/ko/docs/tutorials/stateful-application/zookeeper.md [hunk 6]
```diff
@@ -15,17 +15,17 @@ weight: 40
...
+- [클러스터 DNS](/ko/docs/concepts/services-networking/dns-pod-service/)
+- [헤드리스 서비스](/ko/docs/concepts/services-networking/service/#헤드리스-headless-서비스)
+- [퍼시스턴트볼륨](/ko/docs/concepts/storage/persistent-volumes/)
+- [퍼시스턴트볼륨 프로비저닝](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/persistent-volume-provisioning/)
+- [스테이트풀셋](/ko/docs/concepts/workloads/controllers/statefulset/)
+- [PodDisruptionBudget](/ko/docs/concepts/workloads/pods/disruptions/#파드-disruption-budgets)
+- [파드안티어피니티](/ko/docs/concepts/scheduling-eviction/assign-pod-node/#어피니티-affinity-와-안티-어피니티-anti-affinity)
+- [kubectl CLI](/ko/docs/reference/kubectl/kubectl/)
+
+반드시 최소한 4개의 노드가 있는 클러스터가 필요하며, 각 노드는 적어도 2 개의 CPU와 4 GiB 메모리가 필요하다. 이 튜토리얼에서 클러스터 노드를 통제(cordon)하고 비우게(drain) 할 것이다. **이것은 클러스터를 종료하여 노드의 모든 파드를 퇴출(evict)하는 것으로, 모든 파드는 임시로 언스케줄된다는 의미이다.** 이 튜토리얼을 위해 전용 클러스터를 이용하거나, 다른 테넌트에 간섭을 하는 혼란이 발생하지 않도록 해야 합니다.
```
Comment (pjhwa) [inline content/ko/docs/tutorials/stateful-application/zookeeper.md:28]:
```suggestion
반드시 최소한 4개의 노드가 있는 클러스터가 필요하며, 각 노드는 적어도 2 개의 CPU와 4 GiB 메모리가 필요하다. 이 튜토리얼에서 클러스터 노드를 통제(cordon)하고 비우게(drain) 할 것이다. **이것은 클러스터를 종료하여 노드의 모든 파드를 축출(evict)하는 것으로, 모든 파드는 임시로 언스케줄된다는 의미이다.** 이 튜토리얼을 위해 전용 클러스터를 이용하거나, 다른 테넌트에 간섭을 하는 혼란이 발생하지 않도록 해야 합니다.
```

- 용어집에 있는 용어라 수정했습니다.

**Review by pjhwa — COMMENTED** [review]
@ysyukr 수고 많으셨습니다. 감사합니다!
작은 의견 몇 개 드립니다. 확인하시고 선택적으로 반영하여 주시기 바랍니다~

**Comment by ysyukr** [comment]
@pjhwa 리뷰 감사합니다. 코멘트 확인해서 반영 했습니다. 😄

**Comment by pjhwa** [comment]
감사합니다. 수고하셨습니다~

/lgtm

**Comment by seokho-son** [comment]
@ysyukr @pjhwa 감사합니다!! ^^
/approve

**Pull request merged**